xpathwebdriver/base.py

```diff
@@ -5,16 +5,18 @@
 Copyright (c) 2014, Juju inc.
 Copyright (c) 2011-2013, Joaquin G. Duo
 '''
-import rel_imp; rel_imp.init()
+import rel_imp
+
+rel_imp.init()
 import logging
 import re
 import os
 from .logger import Logger
 from .solve_settings import solve_settings, register_settings
 
 
+# TODO to consider to use PascalCase for this Class or to implement a singleton function based.
 class singleton_decorator(object):
```

Comment:
I was wondering another approach is to implement this function-based singleton decorator and thus avoid linter warnings about the class not using the PascalCase styling convention: 

```
def singleton_decorator(object):    
    instance = [None]
    def wrapper(*args, **kwargs):
        if instance[0] is None:
            instance[0] = object(*args, **kwargs)
        return instance[0]

    return wrapper
```